liblouis/lou_translateString.c

```diff
@@ -219,14 +219,23 @@ findForPassRule(const TranslationTableHeader *table, int pos, int currentPass,
 	return 0;
 }
 
+static widechar
+toLowercase(
+		const TranslationTableHeader *table, const TranslationTableCharacter *character) {
+	if (character->basechar && character->mode == CTC_UpperCase)
+		return ((TranslationTableCharacter *)&table->ruleArea[character->basechar])
+				->value;
+	return character->value;
+}
+
```

Comment:
Not so nice that this function is defined twice (for forward and backards translation)

Reply:
We could move these functions from lou_translateString.c and lou_backTranslateString.c to commonTranslationFunctions.c (and give them a _lou_ prefix and add them to internal.h).